lib/viewing/ViewRelationship.js

```diff
@@ -133,10 +133,7 @@ ViewRelationship.prototype = {
                     break;
 
                 case "html":
-                    while (destinationEl.firstChild) {
-                        destinationEl.removeChild(destinationEl.firstChild);
-                    }
-
+                    destinationEl.innerHTML = "";
```

Comment:
Why not https://developer.mozilla.org/en-US/docs/Web/API/Node/replaceChild

Reply:
replaceChild would only replace one of the nodes vs what I want to do is replace all the nodes

Reply:
I could change back to removing nodes, but If I do, I should at least switch it to remove the last one on each iteration. I think that would be faster